scripts/pipeline/runTest.sh

```diff
@@ -81,3 +82,9 @@ else
     curl -X POST -H 'Content-type: application/json' --data '{"text":"access console at: '$console'"}' $(get_env slack_web_hook_url) </dev/null
     curl -X POST -H 'Content-type: application/json' --data '{"text":"credentials: kubeadmin/'$token'"}' $(get_env slack_web_hook_url) </dev/null
 fi
+
+echo "Cleaning up after tests have be completed"
+echo "switching back to scripts/pipeline directory"
+cd ..
+oc logout
+export CLUSTER_URL=""
```

Comment:
Is it being reset here for the next test run?

Reply:
Yes its resetting for the next run